resources/assets/less/landing.less

```diff
@@ -1,18 +1,22 @@
 @import "./tailwind";
 
-html {
-    @apply .font-sans;
+.blue-overlay:before {
```

Comment:
Hey Jeff!

When I say alphabetically sort properties, I mean this:

```css
.blue-overlay:before {
    background-color: rgba(0, 59, 87, 0.8;
    content: ' ';
    display: block;
```

... rather than what I think you did here, which was sorting the *selectors* alphabetically.